mindone/trainers/train_step.py

```diff
@@ -104,6 +115,10 @@ def construct(self, *inputs):
 
         # 1. compute gradients (of the up-scaled loss w.r.t. the model weights)
         grads = self.grad(self.network, weights)(*inputs, scaling_sens_filled)
+
+        # Gradient communication
+        grads = self.zero_helper.cal_gradients(grads)
```

Comment:
`self.zero_helper` can be None, need to to add a if condition

Reply:
ok